src/trix/controllers/toolbar_controller.coffee

```diff
@@ -135,6 +135,7 @@ class Trix.ToolbarController extends Trix.BasicObject
   setAttribute: (dialogElement) ->
     attributeName = getAttributeName(dialogElement)
     input = getInputForDialog(dialogElement, attributeName)
+    @cleanInputValue(input)
```

Comment:
It would be better to perform this in an `input` event handler so the changes are applied immediately are visible to you as you type.